documentation/content/en/articles/hubs/_index.adoc

```diff
@@ -402,7 +402,7 @@ If Mirror1 does not update for a while, lower tier mirrors will begin to mirror
 [[mirror-official-become]]
 === How to Become Official Then?
 
-We are not accepting any new mirrors at this time.
+Please contact the Source Repository Manager mailto:clusteradm@FreeBSD.org[clusteradm@FreeBSD.org].
```

Comment:
"Source Repository Manager" should be "Cluster Administrators", and maybe a link to "/administration/#t-clusteradm" would be better since it's intended to not use full email.

Reply:
Thanks, I have updated the branch to reflect this change.